cli/cmd/completion.go

```diff
@@ -31,8 +31,8 @@ func newCmdCompletion() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:       "completion [bash|zsh]",
-		Short:     "Shell completion",
-		Long:      "Output completion code for the specified shell (bash or zsh).",
+		Short:     "Output shell completion code for the specified shell (bash or zsh)",
```

Comment:
omg this is exactly 80 chars with the cli indent.

Reply:
In fairness, I stole this from the `kubectl help completion` output.